dGame/dUtilities/VanityUtilities.cpp

```diff
@@ -253,7 +256,7 @@ void ParseXml(const std::string& file) {
 
 			VanityObject objectData {
 				.m_Name = name,
-				.m_LOT = std::stoi(lot),
+				.m_LOT = GeneralUtils::TryParse<LOT>(lot).value_or(LOT_NULL),
```

Comment:
Consider not spawning the object if it is of type lot null

Reply:
will do